xml/System.Windows.Forms.DataVisualization.Charting/LegendItemsCollection.xml

```diff
@@ -95,7 +95,7 @@
 ## Remarks  
  Appends a custom legend item to the collection with the specified text and image to be used as the symbol for the legend item.  
   
- To add a custom legend item that uses a color for its symbol, call the [Add(Color, String)](<xref:System.Windows.Forms.DataVisualization.Charting.LegendItemsCollection.Add(System.Drawing.Color,System.String)) method.  
+ To add a custom legend item that uses a color for its symbol, call the [Add(Color, String)](<xref:System.Windows.Forms.DataVisualization.Charting.LegendItemsCollection.Add(System.Drawing.Color,System.String)>) method.  
```

Comment:
This isn't displaying the proper link text, which is meant to identify the specific method overload. The best alternative is to remove the link text, so that the link is simply: \<xref:System.Windows.Forms.DataVisualization.Charting.LegendItemsCollection.Add(System.Drawing.Color,System.String)>. That should cause the link text to identify the specific overload and the link to go to the specific overload.

Reply:
I assumed it would work, since a similar link works in the other overload of `Add`. But even if it works, plain `<xref>` is better, so I changed both links.